src/libs/Navigation/Navigation.ts

```diff
@@ -442,6 +449,19 @@ function removeScreenFromNavigationState(screen: Screen) {
     });
 }
 
+function removeScreenByKey(key: string) {
```

Comment:
What are we removing it from? Is it from the navigation state? If so I would suggest:

```suggestion
function removeScreenFromNavigationStateByKey(key: string) {
```